lms/djangoapps/support/static/support/jsx/entitlements/data/api/client.js

```diff
@@ -0,0 +1,58 @@
+import 'whatwg-fetch';
+import Cookies from 'js-cookie';
+
+import { entitlementList } from './endpoints';
+
+const HEADERS = {
+  Accept: 'application/json',
+  'Content-Type': 'application/json',
+  'X-CSRFToken': Cookies.get('csrftoken'),
+}
+
+export function requestEntitlements({username}) {
+  return fetch(
+    `${entitlementApi}/?user=${usernameOrEmail}`, {
```

Comment:
[Apparently `fetch` doesn't come with built-in query parameter support](https://github.com/github/fetch/issues/256)

![alt-text](https://media.giphy.com/media/3oz8xIHoB64uYksaLm/giphy.gif)

It's probably overkill, but you could consider using [`qs`](https://www.npmjs.com/package/qs) or [`querystring`](https://www.npmjs.com/package/query-string)